rest/data/task.go

```diff
@@ -16,13 +16,9 @@ import (
 	"github.com/pkg/errors"
 )
 
-// DBTaskConnector is a struct that implements the Task related methods
-// from the Connector through interactions with he backing database.
-type DBTaskConnector struct{}
-
 // FindTaskById uses the service layer's task type to query the backing database for
 // the task with the given taskId.
-func (tc *DBTaskConnector) FindTaskById(taskId string) (*task.Task, error) {
+func FindTaskById(taskId string) (*task.Task, error) {
```

Comment:
These data functions that just wrap the model functions directly should just be removed, and we should call the model function directly.